src/main/resources/assets/nucleus/messages.properties

```diff
@@ -163,6 +164,19 @@ config.commandlogger.list=A comma separated list of commands in the blacklist or
 
 config.mute.blocked=Commands to block when muted. This is in addition to chat and /m already being blocked.
 
+config.mute.maxmutelength=The maximum length a mute may last for (In seconds) unless the user handing the mute has a bypass permission, set to -1 for no maximum.
+config.tempban.maxtempbanlength=The maximum length a temporary ban may last for (In seconds) unless the user handing the mute has a bypass permission, set to -1 for no maximum.
+
+config.warn.actioncommand=The command to execute when a player has a specific number of warnings.
+config.warn.showonlogin=If true, players will be shown all their warnings when they login. If false, they will not.
+config.warn.expire=If true, a record of players warnings will be kept as 'expired warnings'. If false, they will not.
```

Comment:
I think the single quotes will get stripped because of the message formatter - use double quotes or double up on single quotes.